Libraries/Lists/VirtualizedList.js

```diff
@@ -2293,7 +2294,6 @@ class CellRenderer extends React.Component<
         {itemSeparator}
       </CellRendererComponent>
     );
-    // TODO(macOS GH#774)]
```

Comment:
Just wanted to check that this was meant to be removed?

Reply:
I guess so.. since I can't find an opening tag. I might have just been cleaning that up.